fabric-api-dev/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,28 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-api-dev",
+  "name": "Fabric API Dev Utilities",
```

Comment:
Im not too keen on this module name, although I dont really have a better suggestion. I dont think there is a reason to prevent people from enabling these options in prod.

I dont think it fits into any existing module.

Reply:
I'm kinda trying to describe it like, "This is a module that generally adds stuff only useful for developers"